troubleshooting/out-of-memory.md

```diff
@@ -35,13 +35,13 @@ sudo yum clean all
 qvm-remove <VMname>
 ~~~
 
-With this method you lose one VM data, but it'll more securely work.
+With this method you lose the data of one VM, but it'll more reliably work.
```

Comment:
It's a good point, I'm going to focus on grammar/typo for this PR just for scope purposes though.